packages/plugin-essentials/sources/commands/config/get.ts

```diff
@@ -19,6 +36,45 @@ export default class ConfigSetCommand extends BaseCommand {
     if (typeof setting === `undefined`)
       throw new UsageError(`Couldn't find a configuration settings named "${this.name}"`);
 
-    this.context.stdout.write(`${configuration.get(this.name)}\n`);
+    const value = configuration.get(this.name);
+
+    const report = await StreamReport.start({
+      configuration,
+      includeFooter: false,
+      json: this.json,
+      stdout: this.context.stdout,
+    }, async report => {
+      report.reportJson(convertMapsToObjects(value));
+    });
+
+    if (!this.json) {
+      // @ts-ignore: The Node typings forgot one field
+      inspect.styles.name = `cyan`;
+
+      this.context.stdout.write(`${inspect(value, {
+        depth: Infinity,
+        colors: true,
+        compact: false,
+      })}\n`);
+    }
+
+    return report.exitCode();
   }
 }
+
+/**
+ * Converts `Maps` to `Objects` recursively.
```

Comment:
```suggestion
 * Converts `Maps` to `Objects` recursively.
 * todo: Replace by `Object.fromEntries` once we drop support for 10.x (~April 2021) 
```

Reply:
Um, this function currently uses `Object.fromEntries`. Can't we use a polyfill for now? Also, `Object.fromEntries` isn't recursive.

Reply:
I missed the recursive part 😄 Then we should use a polyfill to support Node 10, yep 👍

Reply:
Done.👍

Reply:
I changed the polyfill to `fromentries`, as `object.fromentries` (the "Official ES proposal-compliant polyfill for Object.fromEntries") seems overkill (too many dependencies, as it was made to support ES3).